cms/djangoapps/contentstore/views/preview.py

```diff
@@ -112,6 +112,12 @@ def applicable_aside_types(self, block):
             if aside_type != 'acid_aside'
         ]
 
+    def render(self, block, view_name, context=None):
+        if not context.get('show_preview', True):
+            return self.wrap_xblock(block, view_name, Fragment(), context)
```

Comment:
Nit: I think this could do with an explanatory comment. From what I can tell, this is creating an empty HTML fragment as a placeholder instead of the XBlock content, and then wrapping it with the normal studio XBlock wrapper <div> that contains the header, edit buttons, etc., right?